plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -104,17 +102,10 @@ func (b *backend) pathVenafiSign(ctx context.Context, req *logical.Request, data
 func (b *backend) pathVenafiCertObtain(ctx context.Context, req *logical.Request, data *framework.FieldData, role *roleEntry, signCSR bool) (
 	*logical.Response, error) {
 
-	// When utilizing performance standbys in Vault Enterprise, this forces the call to be redirected to the primary since
-	// a storage call is made after the API calls to issue the certificate.  This prevents the certificate from being
-	// issued twice in this scenario.
-	if (role.StoreByCN || role.StoreBySerial) && b.System().ReplicationState().HasState(consts.ReplicationPerformanceStandby) {
```

Comment:
The reason I stripped off this check is, by default, Vault already forwards all requests that make changes to the primary and they only get executed there. Since `pathVenafiCertObtain` is already _not_ behind a `Read` method, this should just happen automatically for you without this logic. Also, there's some risk in checking against the `const` reflecting replication states. They do tend to change over time.

Reply:
Hmm. the thing is that this code is from PR of another Hashicorp developer - https://github.com/Venafi/vault-pki-backend-venafi/pull/34. We don't have enterprise cluster for now to test both variants. So I'm in doubts :)

Reply:
Interesting! I actually wasn't aware that had changed. Yes, let's keep it. I'm adding it back.

Reply:
Restored in https://github.com/Venafi/vault-pki-backend-venafi/pull/35/commits/6cfc735a27a7635ad8c067fe8321e31287b21583